bofire/utils/entmoot.py

```diff
@@ -0,0 +1,151 @@
+from typing import Tuple
```

Comment:
One can also think about putting this all into `strategies/predictives/enting.py` as these utilities will only used by this one strategy.

What do you think?

Reply:
Yes that makes sense. The file ends up being ~300 lines which isn't so bad.

Reply:
Ok, then let's move it just to one file for now.